packages/wonder-blocks-modal/components/close-button.js

```diff
@@ -2,50 +2,39 @@
 import * as React from "react";
 import {icons} from "@khanacademy/wonder-blocks-icon";
 import IconButton from "@khanacademy/wonder-blocks-icon-button";
+import type {StyleType} from "@khanacademy/wonder-blocks-core";
 
 import ModalContext from "./modal-context.js";
 
 type Props = {|
-    /**
-     * Whether the button is on a dark/colored background.
-     *
-     * Sets primary button background color to white, and secondary and
-     * tertiary button title to color.
-     */
-    light: boolean,
-
-    /**
-     * Called when the close button is clicked.
-     */
-    onClick?: () => void,
+    light?: boolean,
+    onClose?: () => void,
```

Comment:
Where are the prop comments and why the change to `onClose` from `onClick`. The `CloseButton` doesn't get closed, so `onClose` doesn't really make sense

Reply:
Good call.